redisinsight/menu.ts

```diff
@@ -120,7 +120,7 @@ export default class MenuBuilder {
           label: 'New Window',
           accelerator: 'Command+N',
           click: () => {
-            createWindow();
+            createWindow(null);
```

Comment:
Maybe better to set default value for splash?
`splash: BrowserWindow | null = null`

Reply:
yes thanks
changed